modules/understand-etcd-performance.adoc

```diff
@@ -0,0 +1,124 @@
+----
+
+When you use cluster deployments that span many data centers that are using disks for etcd that do not meet the suggested latency, it increases the chances of service-affecting failures and dramatically reduces the network latency that the control plane can sustain.
+
+.Effects of network latency and jitter on etcd
+
+Use the tools that are described in the maximum transmission unit (MTU) discovery and validation section to obtain the average and maximum network latency.
+
+The value of the heartbeat interval should be around the maximum of the average round-trip time (RTT) between members, normally around 1.5x the round-trip time. With the {product-title} default heartbeat interval of 100ms, the suggested RTT between control plane nodes is less than 33 ms with a maximum of less than 66 ms (66 ms x 1.5 = 99 ms). Any network latency that is larger might cause service-affecting events and cluster instability.
+
+The network latency is determined by factors that include the technology of the transport networks, such as copper, fiber, wireless, or satellite, the number and quality of the network devices in the transport network, and other factors. 
+
+Consider network latency with network jitter for exact calculations. Network jitter is the variance in network latency or the variation in the delay of received packets. In efficient network conditions, the jitter should be zero. Network jitter affects the network latency calculations for etcd because the actual network latency over time will be the RTT plus or minus Jitter. For example, a network with a maximum latency of 80 ms and jitter of 30 ms will experience latencies of 110 ms, which means etcd will miss heartbeats. This condition results in request timeouts and temporary leader loss. During a leader loss and re-election, the Kubernetes API cannot process any request that causes a service-affecting event and instability of the cluster.
+
+.Effects of consensus latency on etcd
+
+The procedure can run only on an active cluster. The disk or network test should be completed while you plan a cluster deployment. That test validates and monitors cluster health after a deployment.
+
+By using the `etcdctl` CLI, you can watch the latency for reaching consensus as experienced by etcd. You must identify one of the etcd pods and then retrieve the endpoint health.
+
+.etcd peer round trip time impacts on performance
+
+The etcd peer round trip time is not the same as the network round trip time. This calculation is an end-to-end test metric about how quickly  replication can occur among members.
+
+The etcd peer round trip time is the metric that shows the latency of etcd to finish replicating a client request among all the etcd members.
+The {product-title} console provides dashboards to visualize the various etcd metrics. From the console, select *Observe* then *Dashboards*. Select `etcd` from the dropdown list.
```

Comment:
```suggestion
The {product-title} console provides dashboards to visualize the various etcd metrics. In the console, click *Observe* -> *Dashboards*. From the dropdown list, select *etcd*.
```
OCP docs guidelines: https://www.ibm.com/docs/en/ibm-style?topic=punctuation-symbols#individual-punctuation-marks-or-special-characters

Reply:
@lahinson Should the navigation be: *Observe* > *Dashboards* according to this [page](https://spaces.redhat.com/display/contentservices/UI+guidelines%3A+quick+reference)?
![image](https://github.com/user-attachments/assets/c8cac2df-42bd-45e4-b822-ef368c3978d3)

Reply:
@tmalove Good question! According to that page, the SSG supersedes IBM Style, and I believe the [OCP docs guide](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc) supercedes the SSG. The OCP docs guidelines say to use `->`. The UI elements table that you've referenced is from IBM Style, so I think `->` is preferred in OCP docs over `>`.